django/contrib/auth/hashers.py

```diff
@@ -322,53 +323,76 @@ def encode(self, password, salt):
             parallelism=self.parallelism,
             hash_len=argon2.DEFAULT_HASH_LENGTH,
             type=argon2.low_level.Type.I,
+            version=self.version,
```

Comment:
Ok.  How to access `argon2.low_level.ARGON2_VERSION` without forcing a load?  I could turn `version` into a property that fetches `ARGON2_VERSION` on demand.